test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/ValidationSummaryTagHelperTest.cs

```diff
@@ -288,9 +408,9 @@ public static TheoryData<ModelStateDictionary> ProcessAsync_GeneratesExpectedOut
             // Arrange
             var tagBuilder = new TagBuilder("span2");
             tagBuilder.InnerHtml.SetHtmlContent("New HTML");
+            tagBuilder.Attributes.Add("anything", "something");
             tagBuilder.Attributes.Add("data-foo", "bar");
             tagBuilder.Attributes.Add("data-hello", "world");
-            tagBuilder.Attributes.Add("anything", "something");
```

Comment:
Reordered to match order `TagBuilder` uses anyhow. `TagHelperOutput.Attributes` is maintained in order attributes are added while `TagBuilder.Attributes` maintains alphabetic order.